api/helpers/shared-field-options.js

```diff
@@ -470,4 +470,5 @@ module.exports = {
     "partial_editing",
     "complete",
   ],
+  query: []
```

Comment:
i'm not sure about this solution. this file is for storing the option keys for fields, and `query` isn't a field. i'm wondering if there is another solution here?

Reply:
@ascott thanks! how about this solution? https://github.com/participedia/api/pull/824/commits/43d3b5f4d5ff5c4d66eab032cc9dd4f601059aaf